src/PresentationalComponents/Filters/GroupsFilter.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable camelcase */
+import { useEffect, useState } from 'react';
+import { getSystemsGroups } from '../../Utilities/api';
+
+const useGroupsFilter = (currentFilter = '', apply) => {
+    const [groups, setSystemsGroups] = useState([]);
+
+    useEffect(() => {
+        /* explicitly request OS versions from API */
```

Comment:
Seems this comment needs to be changed ?

Reply:
Yep, removed